### ISSUE TITLE
Kaggle 데이터 분석해보기

### ISSUE THREAD

**Issue by mjaysonnn** [kmu-bigdata/instacart#1]
products.csv

orders.csv   중점으로 분석해보기

**Comment by mjaysonnn** [comment]
Orders_CSV-> order_product 1 (test)

**Comment by mjaysonnn** [comment]
첫번째 미팅 정리

https://www.evernote.com/l/AuG15ZBHH3pKJJSiGdKEIqxktUn99_Junso

**Comment by jinusss** [comment]
https://gist.github.com/jeremystan/c3b39d947d9b88b3ccff3147dbcf6c6b
데이터 디스크립션입니다.
의미가 틀린부분은 업데이트 해주세요.

orders.csv
* order_id: 주문아이디
* user_id: 고객아이디
* eval_set: 주문 평가 set(prior, train, test)
    - prior: 유저가 이전에 주문한 데이터
    - train: 참가자들(대회)을 위한 트레이닝 데이터
    - test: 머신러닝 학습을 위한 테스트 데이터, 예측이 필요한 데이터
* order_number: 주문 수(횟수느낌, 1이면 1번 n이면 n번)
* order_dow: 주문한 요일(0이 일요일부터인지, 월요일부터인지는 불확실)
* order_hour_of_day: 주문 시간(24시 표기법)
* days_since_prior:  마지막 날로부터 몇일 지났는가..?(상한선 30일)

products.csv
* product_id: 품목 아이디
* product_name: 품목 이름
* aisle_id: 외래키
* department_id: 외래키

 aisles.csv
* aisle_id: 마켓의 통로(?) 아이디
* aisle: 통로 이름

departments.csv
* department_id: 매장의 코너 아이디
* department: 코너 이름

order_product_SET.csv
* order_id: 외래키
* product_id: 외래키
* add_to_cart_order: 각각의 품목이 cart에 추가된 순서(?)
* reordered: 1이면 과거에 구매했던 품목, 0이면 그외

**Comment by dongwoLee** [comment]
Test set 주문에 대해서만 재주문을 예측해야한다 .
orders.csv 의 days_since_prior_order는 전 주문을 하고 나서 얼마나 흘렀는지 일 수. (30일로 제한을 둠)
training data로 모델만들고 test데이터와 비교하면 되는데 prior는 왜있는지 모르겠음.ㅠㅠ

**Comment by Jinwoo00** [comment]
> 

>
